src/simulator_core/entities/simulation_configuration.py

```diff
@@ -23,5 +23,6 @@
 class SimulationConfiguration:
```

Comment:
Resolve the space error """ Configuration...

Reply:
Done